File: ios/BFExplorer/Plaoc/IPCCore/IPC/IpcStreamEnd.swift
//
//  IpcStreamEnd.swift
//  IPC
//
//  Created by ui03 on 2023/2/13.
//

import UIKit

struct IpcStreamEnd {

    let type = IPC_DATA_TYPE.STREAM_END
    private(set) var stream_id: String = ""
    
    init(stream_id: String) {
        
        self.stream_id = stream_id
    }
}

extension IpcStreamEnd: IpcMessage {
    init() {
        
    }
}

File: desktop-dev/src/core/ipc/stream/IpcStreamEnd.ts
import { IPC_MESSAGE_TYPE } from "../helper/const.ts";
import { IpcMessage } from "../ipc-message/IpcMessage.ts";

export class IpcStreamEnd extends IpcMessage<IPC_MESSAGE_TYPE.STREAM_END> {
  constructor(readonly stream_id: string) {
    super(IPC_MESSAGE_TYPE.STREAM_END);
  }
}
